src/arvfakecamera.h

```diff
@@ -134,6 +134,11 @@ void		arv_fake_camera_set_fill_pattern	(ArvFakeCamera *camera,
 							 void *fill_pattern_data);
 void 		arv_fake_camera_set_trigger_frequency 	(ArvFakeCamera *camera, double frequency);
 
+/**
```

Comment:
I welcome documentation changes, but that should be in another commit.

Please refer to the gtk-doc documentation: https://developer.gnome.org/gtk-doc-manual/unstable/documenting_syntax.html.en

Reply:
Sorry - yes that should not be here. And as discussed in a previous PR, it needs to go in the .c file :).